pyqtorch/gpsr.py

```diff
@@ -161,9 +158,64 @@ def single_gap_shift(
                 / (4 * torch.sin(spectral_gap * shift / 2))
             )
 
-        def multi_gap_shift(*args, **kwargs) -> Tensor:
-            """Implements multi gap PSR rule."""
-            raise NotImplementedError("Multi-gap is not yet supported.")
+        def multi_gap_shift(
```

Comment:
What about the support of gpu device? In `qadence` we fixed that recently.

Reply:
Thanks for pointing it out @vytautas-a, I added this just now.